prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,58 @@
+"""
+CP1404 Prac 9 - Taxi Simulator
+"""
+
+from prac_09.taxi import Taxi
+from prac_09.silver_service_taxi import SilverServiceTaxi
+
+MENU = "q)uit, c)hoose taxi, d)rive"
+VALID_CHOICES = "qcd"
+
+
+def main():
+    bill = 0
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
+    chosen_taxi = None
+    print(MENU)
+    choice = input(">>> ").lower()
+    while choice not in VALID_CHOICES:
```

Comment:
Try revisiting menu best practices. Although this works there are better options than VALID CHOICES. e.g. while choice != "q".